packages/compiler/src/server/compile-service.ts

```diff
@@ -174,12 +176,30 @@ export function createCompileService({
       if (serverHost.throwInternalErrors) {
         throw err;
       }
+
+      let uri = document.uri;
+      let range = Range.create(0, 0, 0, 0);
+      if (err.info.kind === "emitter" && configFilePath) {
+        const sourceFile = await serverHost.compilerHost.readFile(configFilePath);
+        const emitterName = err.info.metadata.name;
+        const lineAndChar = sourceFile.getLineAndCharacterOfPosition(
+          sourceFile.text.indexOf(emitterName),
```

Comment:
can't we use yaml to load the doc and get the pos? indexof may not return the right string if user has that text in other places